lib/omnibus/packagers/deb.rb

```diff
@@ -676,33 +676,42 @@ def remove(packages)
       shellout!("apt-get remove -y --force-yes #{packages}")
     end
 
-    def compression_level(val = nil)
-      unless val.nil?
+    def compression_level(val = NULL)
+      if null?(val)
```

Comment:
Is there any specific reason you turned from `nil` to `NULL` (I think I understand the logic was reversed and it might be enough to change the behaviour, isn't it?)

Reply:
Mostly consistency with the rest of the code, I honestly have no idea if it makes a difference